lib/rubocop/cop/rspec/pending_without_reason.rb

```diff
@@ -0,0 +1,123 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module RSpec
+      # Checks for pending or skipped examples without reason.
+      #
+      # @example
+      #   # bad
+      #   pending 'does something' do
+      #   end
+      #
+      #   # bad
+      #   it 'does something', :pending do
+      #   end
+      #
+      #   # bad
+      #   it 'does something' do
+      #     pending
+      #   end
+      #
+      #   # bad
+      #   xdescribe 'something' do
+      #   end
+      #
+      #   # bad
+      #   skip 'does something' do
+      #   end
+      #
+      #   # bad
+      #   it 'does something', :skip do
+      #   end
```

Comment:
How about `it 'works on newer Ruby versions', skip: RUBY_VERSION < 3.0` do?

Reply:
Ideally, we would like to make it an offense, but in this case the offense is not detectable. In the current implementation, it is only considered an offense when the metadata is specified in Symbol or Boolean literal.